tests/integration/conftest.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Fixtures for Jenkins-k8s-operator charm integration tests."""
+
+import pytest
+import pytest_asyncio
+from juju.application import Application
+from juju.client._definitions import FullStatus
+from juju.model import Model
+from pytest import FixtureRequest
+from pytest_operator.plugin import OpsTest
+
+
+@pytest.fixture(scope="module", name="model")
+def model_fixture(ops_test: OpsTest) -> Model:
+    """The testing model."""
+    assert ops_test.model
+    return ops_test.model
+
+
+@pytest.fixture(scope="module", name="jenkins_image")
+def jenkins_image_fixture(request: FixtureRequest) -> str:
+    """The OCI image for Jenkins charm."""
+    return request.config.getoption("--jenkins-image")
```

Comment:
How about raise an error here and provide some useful error messages here if the Jenkins image is not provided by command line arguments.

Reply:
Updated thanks!